kie-integration-test-coverage/kie-maven-plugin-default-execmodel-tests/kjar-with-instrumentation-default-execmodel-wrapper/pom.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <groupId>org.drools.testcoverage</groupId>
+    <artifactId>kie-maven-plugin-default-execmodel-tests-parent</artifactId>
+    <version>7.44.0-SNAPSHOT</version>
```

Comment:
Should this be aligned with 7.45.0-SNAPSHOT now?

Reply:
+1